### ISSUE TITLE
import list.js instead of defining the script source from the template

### ISSUE THREAD

**Issue by rainercedric23** [javve/list.js#542]
Hi do you have any idea how can I import the list and instantiating it inside the JS? on this use case I am using handle bars for my js templating engine. 

I have this line of code:

`

    import List from "BaseVendor/list";
    import viewTemplate from "SiteTemplate/handlebars/view-template.handlebars";

    export default function view() {
        var $this = this,
            $wrapper = document.getElementById('view-container'),
            $loader = document.getElementById('view-loader');

         this.init = function() {
            template = viewTemplate ();
            $wrapper.innerHTML = template;
    
              var options = {
                   valueNames: [
                        'name',
                        'date'
                   ],
                   pagination: true,
                   page: 2,
              };

             var viewList = new List('view-list', options);
        }
    }

`

Basically the template had the html view list div. Currently I can't use the new List to work.

If you can point out what I am doing wrong here that will really help. thanks.

**Comment by rainercedric23** [comment]
While further investigating the issue, I've found that If I import the list.js thru `import List from "BaseVendor/list";` it just returns an empty object whereas the other vendor libraries return an object where you can use its functionalities.

Creating an additional webpack entry in my app also causes an issue (parsing all vendor libraries into one file) because it encloses it in one webpack entry array key that can be accessed by other js components( __webpack_require__(n) ).

**Comment by rainercedric23** [comment]
I've fixed the webpack compile issue thru adding an extra line of code at the end of file `export default List;`

**Comment by michaelwoodew** [comment]
I'm having the same issue importing into Aurelia...
How can list.js be imported as a webpack module?

**Comment by rainercedric23** [comment]
hi @michaelwoodew please see the comment above your question, that is how I solve it.
